app/code/Magento/VaultGraphQl/Model/Resolver/PaymentTokens.php

```diff
@@ -19,18 +20,44 @@
  */
 class PaymentTokens implements ResolverInterface
 {
+    /**
+     * Cart types
+     */
+    const BASE_CART_TYPES = [
+        'VI' => 'Visa',
+        'MC' => 'MasterCard',
+        'AE' => 'American Express',
+        'DN' => 'Diners',
+        'DI' => 'Discover',
+        'JCB' => 'JCB',
+        'UN' => 'UnionPay',
+        'MI' => 'Maestro International',
+        'MD' => 'Maestro Domestic',
+        'HC' => 'Hipercard',
+        'ELO' => 'Elo',
+        'AU' => 'Aura'
+    ];
+
     /**
      * @var PaymentTokenManagement
      */
     private $paymentTokenManagement;
 
+    /**
+     * @var JsonSerializer
+     */
+    private $serializer;
+
     /**
      * @param PaymentTokenManagement $paymentTokenManagement
+     * @param JsonSerializer $serializer
      */
     public function __construct(
-        PaymentTokenManagement $paymentTokenManagement
+        PaymentTokenManagement $paymentTokenManagement,
+        JsonSerializer $serializer
```

Comment:
Hello @Usik2203,
could you please add the new dependency in a backward-compatible way?
https://devdocs.magento.com/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter

```suggestion
        JsonSerializer $serializer = null
```